web/src/components/filings-calendar/FilingsCalendarAsList.tsx

```diff
@@ -108,6 +131,17 @@ export const FilingsCalendarAsList = (props: Props): ReactElement => {
                     />
                   );
                 }
+                if (event.calendarEventType === "XRAY") {
+                  return (
+                    <CalendarEventItem
+                      key={"XRAY_RENEWAL"}
```

Comment:
dust - im suprised the IDE is not screaming at you for using a static value for the key.